Move CrossGuard Get Started in the nav (#354)
pulumi/pulumi-hugo: themes/default/content/docs/guides/crossguard/get-started.md

           and security compliance when creating cloud resources.
linktitle: Get Started
weight: 1
aliases: [
    "/docs/get-started/policy-as-code/",
    "/docs/get-started/crossguard/",
    "/docs/get-started/crossguard/authoring-a-policy-pack/",
    "/docs/get-started/crossguard/enforcing-a-policy-pack/"
]

menu:
  userguides:
    parent: crossguard
---

Pulumi CrossGuard is a product that provides gated deployments via Policy as Code.

Often organizations want to empower developers to manage their infrastructure yet are concerned about giving them full access. CrossGuard allows administrators to provide autonomy to their developers while ensuring compliance to defined organization policies.

Using Policy as Code, users can express business or security rules as functions that are executed against resources in their stacks. Then using CrossGuard, organization administrators can apply these rules to particular stacks within their organization. When policies are executed as part of your Pulumi deployments, any violation will gate or block that update from proceeding.

> Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. More information on language support for policies is [here]({{< relref "/docs/guides/crossguard#languages" >}}).

Reviewer: ```suggestion
Policies can be written in TypeScript/JavaScript (Node.js) or Python and can be applied to Pulumi stacks written in any language. Learn more about [language support for policies]({{< relref "/docs/guides/crossguard#languages" >}}).
```

been trying really hard to get us to stop using here, click here, learn more links
ref: https://webaim.org/techniques/hypertext/link_text#text

Reviewer 2: > been trying really hard to get us to stop using here, click here, learn more, etc

FWIW, though, I don't believe we have a doc anywhere explaining this kind of docs-style guidance. When we do, we might want to make reviewing part of a PR template. I'll get something filed if there isn't already.

Author: Absolutely, good chance. Note that this wasn't new -- just moved content around -- but I agree with the feedback so have applied. I'd love if we could document the guidance, per @cnunciato's suggestion, and automatically lint for this!
